lib/src/common/repos_service.dart

```diff
@@ -309,16 +329,17 @@ class RepositoriesService extends Service {
     return false;
   }
 
+  /// https://docs.github.com/en/rest/collaborators/collaborators#add-a-repository-collaborator
   Future<bool> addCollaborator(RepositorySlug slug, String user) async {
     ArgumentError.checkNotNull(slug);
     ArgumentError.checkNotNull(user);
     return github
         .request(
           'PUT',
           '/repos/${slug.fullName}/collaborators/$user',
-          statusCode: StatusCodes.NO_CONTENT,
+          statusCode: StatusCodes.CREATED,
```

Comment:
And we might even want to remove this statusCodeCheck since one or the other scenario will fail if we can only provide 1 successful code?
```suggestion
```

Reply:
I'm sorry about the delay. I completely forgot this PR.